src/ArrayConnection.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
I'm not sure that we need this as a part of this library as it's kind of a hack?